src/Altinn.Platform/Altinn.Platform.Profile/Profile/Configuration/GeneralSettings.cs

```diff
@@ -19,5 +19,10 @@ public class GeneralSettings
         /// Name of the cookie for where JWT is stored
         /// </summary>
         public string JwtCookieName { get; set; }
+
+        /// <summary>
+        /// The number of seconds the user profile will be kept in the cache
+        /// </summary>
+        public int ProfileCacheLifetimeSeconds { get; set; } = 600;
```

Comment:
Nothing to do with your change as such, but would consider renaming the settings/options to something more specific to what it should cover. With general as a name anything can go into that class. Naming it BridgeSettings (or something similar) would make it clearer and avoid other devs adding settings that shouldn't be grouped together.